### PR TITLE
added the max rectangle code with changes

### PR DESCRIPTION
**Pull request by shraddhavp**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

yes

### Description

added the code for max_sum rectangle with recent changes requested

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#937

### CODE REVIEW

**Comment by shraddhavp** [comment]
Changed the PR please check it and let me know for changes 
@iamrajiv

File: Python/cp/max_sum.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+    for left in range(cols):
+        temp = [0 for _ in range(rows)]
+        for right in range(left, cols):
+            for i in range(rows):
+                temp[i] += rectangle[i][right]
+
+            kadanes_result = kadanes(temp)
+            if kadanes_result.maxSum > result.maxSum:
+                result = Result(kadanes_result.maxSum, left, right, kadanes_result.start, kadanes_result.end)
```
Comment (iamrajiv) [inline Python/cp/max_sum.py]: line too long (109 > 79 characters)

File: Python/cp/max_sum.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+if __name__ == '__main__':
+    rectangle = [[2,  1, -3, -4,  5],
+                 [0,  6,  3,  4,  1],
+                 [2, -2, -1,  4, -5],
+                 [-3,  3,  1,  0,  3]]
+
+    result = max_sub_sub_rectangle(rectangle)
+    assert 18 == result.maxSum
+    print (result)
+    # Time complexity: O(n^3) or O(COL^2*ROW)
```
Comment (iamrajiv) [inline Python/cp/max_sum.py:58]: update this to ` #Time complexity: O(n^3) or O(COL^2*ROW)`

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@shraddhavp Remove unnecessary commits related to `ShellSort.cpp`. There should be only 2 files changed i.e. README and program but I can see there is 4 file changed please take a look.

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@shraddhavp please remove the unnecessary changes like deletion of `ShellSort.cpp` from this PR

**Pull request closed without merging**